src/components/NetworkTabs.tsx

```diff
@@ -0,0 +1,68 @@
+import { TabList } from "@mui/lab";
+import { NoSsr, Tab } from "@mui/material";
+import isEqual from "lodash/isEqual";
+import {
+  defaultStreamQueryOrdering,
+  defaultStreamQueryPaging,
+} from "../components/NetworkStreams";
+import { track } from "../hooks/useMatomo";
+import { useAppSelector } from "../redux/hooks";
+import { networks } from "../redux/networks";
+import { sfSubgraph } from "../redux/store";
+import NetworkDisplay from "./NetworkDisplay";
+
+type NetworkTabsProps = {
+  activeTab: string;
+  setActiveTab: (...args: any[]) => void;
+};
+
+const NetworkTabs: React.FC<NetworkTabsProps> = ({ setActiveTab }) => {
+  const prefetchStreamsQuery = sfSubgraph.usePrefetch("streams", {
+    ifOlderThan: 45,
+  });
+
+  const displayedTestnetChainIds = useAppSelector(
+    (state) =>
+      Object.entries(state.appPreferences.displayedTestNets)
+        .filter(([_, isDisplayed]) => isDisplayed)
+        .map(([chainId]) => Number(chainId)),
+    isEqual
+  );
+
+  return (
+    <NoSsr>
+      <TabList
+        variant="scrollable"
+        scrollButtons="auto"
+        data-cy={"landing-page-networks"}
+        onChange={track("network-tab-change", (_event, newValue: string) =>
```

Comment:
I think the page it's being triggered on is attached to the event, so this should be okay.